haystack/database/elasticsearch.py

```diff
@@ -442,25 +442,8 @@ def update_embeddings(self, retriever: BaseRetriever, index: Optional[str] = Non
             raise RuntimeError("Specify the arg `embedding_field` when initializing ElasticsearchDocumentStore()")
 
         docs = self.get_all_documents(index)
-        passages = [d.text for d in docs]
-    
-        #TODO Index embeddings every X batches to avoid OOM for huge document collections
```

Comment:
Hi @bogdankostic,
 I think this TODO shouldn't be deleted. This could be a great feature for a future Haystack release.

Reply:
Of course, this shouldn't be deleted. Thanks for noticing that! Will put it back in with the next commit. :)